src/app/views/App.js

```diff
@@ -1,141 +1,145 @@
-import React from 'react';
-import { Client4 } from 'mattermost-redux/client';
+import { Client4 } from "mattermost-redux/client";
+import React, { useEffect, useState } from "react";
+import browser from "webextension-polyfill";
 
 // Set this to your Mattermost SiteURL.
-const MATTERMOST_URL = 'http://localhost:8065';
```

Comment:
I realize there's no eslint configured for the repo but our style guide uses the single quotes `'` so it'd be preferable to stick with those to be consistent

Here's the eslint we follow in other repos https://github.com/mattermost/mattermost-webapp/blob/master/.eslintrc.json

Reply:
I've integrated eslint and mattermost configuration with minor tweaks to accommodate this project